### ISSUE TITLE
Don't show warning that isn't related to the scanned files

### ISSUE THREAD

**Issue by php-coder** [svenkreiss/html5validator#34]
Recently TravisCI [added `export _JAVA_OPTIONS="-Xmx2048m -Xms512m"`](https://docs.travis-ci.com/user/build-environment-updates/2017-09-06/#Added) to their environment and my tests started to fail because html5validator is showing warning for that:

```console
$ export _JAVA_OPTIONS="-Xmx2048m -Xms512m"
$ html5validator --root src/main/webapp/WEB-INF/views
WARNING:html5validator.validator:Picked up _JAVA_OPTIONS: -Xmx2048m -Xms512m
$ echo rc=$?
rc=1
```

While I'm able to ignore it with `--ignore-re` I believe that this is a bug and should be fixed.

**Comment by php-coder** [comment]
Note that this warning is shown even without specifying `--show-warnings` option.

**Comment by svenkreiss** [comment]
Ugh. Looking into this.

**Comment by svenkreiss** [comment]
Thanks for reporting this! 
Version 0.2.8 should have this fixed and is on pypi now.
